clang/lib/AST/ExprConstant.cpp

```diff
@@ -4443,7 +4444,7 @@ struct CompoundAssignSubobjectHandler {
       return foundVector(Subobj, SubobjType);
     case APValue::Indeterminate:
       Info.FFDiag(E, diag::note_constexpr_access_uninit)
-          << /*read of=*/0 << /*uninitialized object=*/1;
+          << /*read of=*/0 << /*uninitialized object=*/1 << E->getSourceRange();
```

Comment:
I prefer `E->getLHS()->getSourceRange()` because the uninitialized object is known to exist in LHS.